library/debug.c

```diff
@@ -405,4 +405,44 @@ void mbedtls_debug_printf_ecdh( const mbedtls_ssl_context *ssl, int level,
 }
 #endif /* MBEDTLS_ECDH_C */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+#define BITS_OF( var )    ( sizeof( var ) * 8 )
```

Comment:
```suggestion
```

Reply:
This is still used in line 412. Removal of this macros means we need to hard code the length of `ticket_flag_name_table`.

Reply:
Please remove ln 412. I do not get that. But that really waste space.